controllers/telemetry/telemetrymanager_controller_test.go

```diff
@@ -0,0 +1,60 @@
+package telemetry
+
+import (
+	"context"
+	"fmt"
+	telemetryv1alpha1 "github.com/kyma-project/telemetry-manager/apis/telemetry/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Deploying a TelemetryManager", func() {
+	const (
+		timeout  = time.Second * 100
```

Comment:
I think that this is a typo and you intended to have the timeout as 10 seconds instead of 100 seconds, right?

Reply:
Okay, I have noticed now that in the controller test for LogPipeline and LogParser the timeout is 10s, but in the TracePipeline the timeout is 100s. Is there a reason for this big difference in the timeout?